patches/chrome-android-java-res-layout-bookmark_action_bar.xml.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/res/layout/bookmark_action_bar.xml b/chrome/android/java/res/layout/bookmark_action_bar.xml
+index 8b3a177b0320ec1de7f1e7bf817b4d0aea1861b8..f2ffdf54ee8551769c32aafa97052e137fa38501 100644
+--- a/chrome/android/java/res/layout/bookmark_action_bar.xml
++++ b/chrome/android/java/res/layout/bookmark_action_bar.xml
+@@ -5,7 +5,7 @@ Use of this source code is governed by a BSD-style license that can be
+ found in the LICENSE file.
+ -->
+ 
+-<org.chromium.chrome.browser.bookmarks.BookmarkActionBar
++<org.chromium.chrome.browser.bookmarks.BraveBookmarkActionBar
```

Comment:
I such cases we just override resources by placing file with changes at the same path in `brave` folder.